components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -728,6 +761,80 @@
         </div>
         <button on:click={saveProperties}>Save Editor Options</button>
       </section>
+
+      <section class="booking-details">
+        <h1>Custom Fields</h1>
+        <p class="intro">
+          Ask users to fill out a few details before booking an event with you.
+          By deafult, they will be asked for their name and email address.
+        </p>
+        <div class="contents">
+          {#if customFields}
+            <table cellspacing="0" cellpadding="0">
+              <thead>
+                {#each customFieldKeys as key}
+                  <th>{key}</th>
+                {/each}
+                <th />
+              </thead>
+              <tbody>
+                {#each customFields as field}
+                  <tr>
+                    {#each customFieldKeys as key}
+                      <td>{field[key] || "—"}</td>
+                    {/each}
+                    <td
+                      ><button on:click={() => removeCustomField(field)}
+                        >Remove</button
+                      ></td
+                    >
+                  </tr>
+                {/each}
+                <tr class="add-new">
+                  <td>
+                    <input
+                      type="text"
+                      placeholder="Title"
+                      bind:this={newFieldTitleElement}
+                      bind:value={newCustomField.title}
+                    />
+                  </td>
+                  <td>
+                    <input
+                      type="text"
+                      placeholder="Description"
+                      bind:value={newCustomField.description}
+                    />
+                  </td>
+                  <td>
+                    <select bind:value={newCustomField.type}>
+                      <option value="text">Text</option>
+                      <option value="checkbox">Checkbox</option>
+                    </select>
+                  </td>
```

Comment:
I think these could use some labels or `aria-labelledby` with `id` from the column headers.